src/Components/Counter.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+function Counter({ fruitName, fruitImage, buttonType }) {
+
+    const [fruitCounterValue, setFruitCounterValue] = useState(0);
```

Comment:
Je hebt hier state aangemaakt in je Counter.js component. 

Eigenlijk wil je altijd je state in je App.js aanmaken. Het is namelijk een stuk eenvoudiger om de state dan door te geven via properties aan meerdere componenten. Andersom (van component naar app) is dit veel lastiger.